### ISSUE TITLE
Add disconnect button

### ISSUE THREAD

**Issue by ntraut** [ConnectProject/mavoix-app#15]
Currently there is theoretically no way to disconnect when logged in.

**Comment by morvanr** [comment]
how and where do you want that to be implemented ?

**Comment by r03ert0** [comment]
* It would be nice to have a list in mavoix-panel of all devices that accessed mavoix-app using the same account, with a 'disconnect' button on the side.
* if a devices accesses mavoix-app for the first time, login/password are requested, and the device should display in mavoix-panel.
* if a device accesses mavoix-app for the 2nd time, no login/password is requested and the first tab is directly displayed.
* if in the meantime the device was disconnected from mavoix-panel, and the device tries to access mavoix-app, login and password are requested.

what do you think?

**Comment by r03ert0** [comment]
yes (we don't want the kids disconnecting the app by mistake :) )
